content/11.interpretation.md

```diff
@@ -2,7 +2,7 @@
 
 Once we have trained an accurate deep model, we often want to use it to deduce scientific findings.
 In doing so, we need to take care to correctly interpret the model's predictions.
-We know that the basic tenets of machine learning also apply to deep learning ([Tip 1](#concepts)), but because deep models can be difficult to interpret intuitively, there is a temptation to anthropomorphize the models.
+Because deep models can be difficult to interpret intuitively, there is a temptation to anthropomorphize the models.
```

Comment:
What does it mean to "anthropomorphize the models?" I know the meaning of the term and it's application as a literary device, but not sure of its exact meaning here...

Reply:
I see this happening a lot when folks say the network learns to recognize lines, then a beak, then a bird in neural network features of increasing complexity (or similar). It's hard for me to see this paper and view that statement as consistent with these results: https://distill.pub/2017/feature-visualization/

I agree with @SiminaB we need a reference and example here.

Reply:
Tracked in #268